robocorp-code/src/robocorp_code/inspector/image/_bridge.py

```diff
@@ -0,0 +1,187 @@
+        environ.pop("PYTHONHOME", "")
+        environ.pop("VIRTUAL_ENV", "")
+        environ["PYTHONIOENCODING"] = "utf-8"
+        environ["PYTHONUNBUFFERED"] = "1"
+        cmd = [
+            sys.executable,
+            str(Path(__file__).parent / "_snipping_tool.py"),
+        ]
+        if self.endpoint is not None:
+            self.endpoint.notify("$/imageInspectorState", {"state": STATE_PICKING})
+
+        self.snipping_process = subprocess.Popen(
+            cmd,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            stdin=subprocess.PIPE,
+            env=environ,
+            bufsize=0,
+        )
+        stdout, _ = self.snipping_process.communicate()
+
+        # If you want to capture the output, you can use stdout.decode('utf-8') to convert it to a string
+        output = stdout.decode("utf-8")
+
+        #: Validating the result from the Snipping Tool
+        if not output:
+            if self.endpoint is not None:
+                self.endpoint.notify("$/imageInspectorState", {"state": STATE_CLOSED})
+            raise RuntimeError("Snipping Tool closed abruptly!")
+
+        try:
+            result = json.loads(output)
+        except ValueError as err:
+            if self.endpoint is not None:
+                self.endpoint.notify("$/imageInspectorState", {"state": STATE_CLOSED})
+            raise RuntimeError(f"Malformed response from picker: {err}") from err
+
+        if "error" in result:
+            if self.endpoint is not None:
+                self.endpoint.notify("$/imageInspectorState", {"state": STATE_CLOSED})
+            raise RuntimeError(result["error"])
+
+        self.logger.info("=== Snapshot taken!")
+        if self.endpoint is not None:
+            self.endpoint.notify("$/imageInspectorState", {"state": STATE_CLOSED})
+        return result
+
+    def pick(self, confidence: Optional[int] = None):
+        self.logger.info("=== Starting interactive picker")
+        if self.endpoint is not None:
+            self.endpoint.notify("$/imageInspectorState", {"state": STATE_INITIALIZING})
+        result = self._launch_snipper()
+
+        self.logger.info("=== Snapshot Result:", result)
+
+        return {
+            "screenshot": f"data:image/png;base64,{result['screenshot']}",
+            "screenResolutionWidth": result["screen_resolution_width"],
+            "screenResolutionHeight": result["screen_resolution_height"],
+            "screenPixelRatio": result["screen_pixel_ratio"],
+            "confidence": confidence,
+        }
+
+    def stop(self):
+        self.logger.info("=== Stopping interactive picker...")
+        if self.snipping_process is not None:
+            self.snipping_process.terminate()
+            self.snipping_process.kill()
+        self.logger.info("=== Process stopped")
+
+    def validate(
+        self,
+        image_base64: str,
+        confidence: Optional[int] = None,
+    ):
+        needle = _base64_to_image(image_base64)
```

Comment:
I believe the validation was the really slow part right? (because pyscreeze is pretty slow for that without opencv).

My suggestion would be not even implementing it at this point until we have the proper backend functionality (so, I'd just raise an exception here... you can leave the current implementation commented out as it does have the part related to grabbing monitor info and so forth).